src/posix.mak

```diff
@@ -477,4 +477,14 @@ zip:
 
 #############################
 
+ifneq ($(DOCDIR),)
+html: $(DOCDIR)/.generated
+$(DOCDIR)/.generated: $(DMD_SRCS) $(ROOT_SRCS) $(HOST_DMD_PATH)
+	$(HOST_DMD_RUN) -of- $(MODEL_FLAG) -J. -c -Dd$(DOCDIR)\
+	  $(DFLAGS) $(DOCFMT) $(DMD_SRCS) $(ROOT_SRCS)
+	touch $@
+endif
```

Comment:
Well ergonomy is really not a priority here. This is invoked as part of the dlang.org build.